ttnn/cpp/ttnn/operations/embedding_backward/device/kernels/compute/embedding_backward.cpp

```diff
@@ -38,6 +38,11 @@ void MAIN {
             uint32_t tile_to_get = 0;
             cb_get_tile(cb_mask, tile_to_get, &idx_addr_ptr);
             uint32_t idx_addr = reinterpret_cast<uint32_t>(idx_addr_ptr);
+#if defined ARCH_BLACKHOLE
```

Comment:
```suggestion
#if defined(ARCH_BLACKHOLE)
```
Although both forms are correct and compile, I think this is a preferred one in our codebase. Code linters tend to favor this style too.